### DISCUSSION TITLE
Help! I can't log in and I don't know why

### DISCUSSION THREAD

**Discussion by tearlach95** [chromiecraft/chromiecraft#3580] [General]
Dear Chromiecraft staff & members
Whenever I try to log in (account "tearlach"), I get the following message: "This WoW account has been closed and is no longer available." Some friends have told me that's because I may have been banned, but I'm quite sure there's no explainable reason for that. I hate tricks, and I have never offended or have any argument with any other player. I think this is all a big mistake, maybe related to the fact that when I logged in today (at 1 server time approx) I was very lagged. Thanks for your attention, and I hope to be back soon.

**Comment by 55Honey** [comment]
Hello @tearlach95,
Thanks for reaching out.
Since we want to prevent you from doxxing your account details in public, please join our discord and open a support-ticket. https://discord.gg/ChromieCraft

Please note that we do not accept ban appeals at all on GitHub and any try will result in immediate deletion of the conversation without further response.

Regards

**Reply by tearlach95** [reply]
I will do so, thanks a lot. Greetings.
